CLAUDE.md

```
@@ -351,14 +351,101 @@ This project prioritizes **execution-time trace analysis** over traditional code
+**Result**: Xdebug session established, waiting for MCP commands on port 9004
+
+### Step 2: AI Controls via MCP Tools (NO xdebug_connect needed!)
+```bash
+# AI can directly use these tools on the existing session:
+xdebug_step_into     # Step into function
+xdebug_step_over     # Step over line
+xdebug_get_variables # Get current variables  
+xdebug_eval          # Evaluate expressions
+xdebug_set_breakpoint # Set breakpoints
+```
+
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)